src/components/Users/UserAvatar.tsx

```diff
@@ -19,20 +19,25 @@ import uploadFile from "@/Utils/request/uploadFile";
 import useTanStackQueryInstead from "@/Utils/request/useQuery";
 import { formatDisplayName, sleep } from "@/Utils/utils";
 
-export default function UserAvatar({ username }: { username: string }) {
+export default function UserAvatar({
+  username,
+  refetchUserData,
+}: {
+  username: string;
+  refetchUserData?: () => void;
+}) {
   const { t } = useTranslation();
   const [editAvatar, setEditAvatar] = useState(false);
   const authUser = useAuthUser();
 
-  const {
-    data: userData,
-    loading: isLoading,
-    refetch: refetchUserData,
-  } = useTanStackQueryInstead(routes.getUserDetails, {
-    pathParams: {
-      username: username,
+  const { data: userData, loading: isLoading } = useTanStackQueryInstead(
+    routes.getUserDetails,
+    {
+      pathParams: {
+        username: username,
+      },
```

Comment:
let's switch these to using tanstack query's useQuery instead?

refer: https://docs.ohc.network/docs/care/development/data-fetching-in-care/

Reply:
can be done separatley